diff --git a/third-party/yoga/src/yoga/YGValue.h b/third-party/yoga/src/yoga/YGValue.h
--- a/third-party/yoga/src/yoga/YGValue.h
+++ b/third-party/yoga/src/yoga/YGValue.h
@@ -66 +66 @@
-inline YGValue operator"" _pt(long double value) {
+inline YGValue operator""_pt(long double value) {
@@ -69,2 +69,2 @@
-inline YGValue operator"" _pt(unsigned long long value) {
-  return operator"" _pt(static_cast<long double>(value));
+inline YGValue operator""_pt(unsigned long long value) {
+  return operator""_pt(static_cast<long double>(value));
@@ -73 +73 @@
-inline YGValue operator"" _percent(long double value) {
+inline YGValue operator""_percent(long double value) {
@@ -76,2 +76,2 @@
-inline YGValue operator"" _percent(unsigned long long value) {
-  return operator"" _percent(static_cast<long double>(value));
+inline YGValue operator""_percent(unsigned long long value) {
+  return operator""_percent(static_cast<long double>(value));
